README.md

```diff
@@ -49,6 +49,6 @@ And set this formatter as the new logging formatter:
 
 ```go
 log.Infof("Log messages are now sent to Graylog (udp://%s)", graylogAddr) // Give a hint why logs are empty
-log.AddHook(graylog.NewGraylogHook(graylogAddr, "api", map[string]interface{}{})) // set graylogAddr accordingly
+log.Hooks.Add(graylog.NewGraylogHook(graylogAddr, "api", map[string]interface{}{})) // set graylogAddr accordingly
```

Comment:
hmm, we have another occurence in our Readme, I guess it's a copy/paste from something existing. Do you mind changing this https://github.com/gemnasium/logrus-graylog-hook/blame/master/README.md#L27 as well? And I will merge this PR.
Thanks!